README.md

```diff
@@ -162,225 +189,414 @@ and modify the `system-reserved` line under `kubeletExtraArgs`:
 ./deploy-test-resources.sh
 ```
 
-5. :gear: If Konflux was installed on a cluster hosted in a remote machine, SSH port-forwarding can 
-be used to access. Open an additional terminal and run the following command
-(make sure to add the details of your remote machine and user):
+5. :gear: As a reminder, if you've installed Konflux on a remote host, you will need to configure SSH port
+forwarding to access your cluster. To do this, open an additional terminal and run the following command
+(make sure to add the details for your remote user and IP address):
 
 ```bash
-ssh -L 9443:localhost:9443 $USER@$VM_IP
+ssh -L 9443:localhost:9443 <remote_host_user_name>@<remote_host_ip_address>
 ```
 
-6. The UI will be available at https://localhost:9443. You can login using a test user.
+6. Once you've set up port forwarding, your cluster's UI will be available at https://localhost:9443.
+You can log in using the test user's account.
 
-`username:` `user2@konflux.dev`
+**username**: **user@konflux.dev**
+**password**: **password**
 
-`password:` `password`
-
-We now have Konflux up and running. Next, we shall configure Konflux to respond
-to Pull Request webhooks, build a user application and push it to a registry.
+You now have Konflux up and running. Next, we'll configure Konflux to respond to pull request webhooks,
+build a user application, and push it to a registry.
 
 ## Enable Pipelines Triggering via Webhooks
 
-Pipelines Can be triggered by Pull Request activities, and their outcomes will be
-reported back to the PR page in GitHub.
+You can configure Konflux to trigger pipelines anytime someone submits a pull request.
 
-A GitHub app is required for creating webhooks that Tekton will listen on. When deployed
-in a local environment like Kind, GitHub will not be able to reach a service within the
-cluster. For that reason, we need to use a proxy that will listen on such events
-from within the cluster and will relay those events internally.
+You can also configure Konflux to send the outcomes of these triggers back to your
```

Comment:
Here as well, we need to make sure that this does not come out as an optional step.